docs/api/desktop-capturer.md

```diff
@@ -12,32 +12,30 @@ title is `Electron`:
 // In the renderer process.
 const { desktopCapturer } = require('electron')
 
-desktopCapturer.getSources({ types: ['window', 'screen'] }, async (error, sources) => {
-  if (error) throw error
-  for (const source of sources) {
-    if (source.name === 'Electron') {
-      try {
-        const stream = await navigator.mediaDevices.getUserMedia({
-          audio: false,
-          video: {
-            mandatory: {
-              chromeMediaSource: 'desktop',
-              chromeMediaSourceId: source.id,
-              minWidth: 1280,
-              maxWidth: 1280,
-              minHeight: 720,
-              maxHeight: 720
-            }
+const sources = await desktopCapturer.getSources({ types: ['window', 'screen'] })
```

Comment:
linting step fails here:
`Parsing error: Can not use keyword 'await' outside an async function`